src/main/java/org/team4159/frc/robot/commands/auto/FollowTrajectory.java

```diff
@@ -0,0 +1,44 @@
+package org.team4159.frc.robot.commands.auto;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj2.command.*;
+
+import org.team4159.frc.robot.subsystems.Drivetrain;
+
+import static org.team4159.frc.robot.Constants.*;
+
+public class FollowTrajectory extends SequentialCommandGroup {
+  public FollowTrajectory(Trajectory trajectory, Drivetrain drivetrain) {
+    addRequirements(drivetrain);
+
+    drivetrain.zeroSensors();
```

Comment:
do we always want this or should we just do this before each auto?
sometimes we might want to chain trajectories

Reply:
was temporary, i'll nix it